code/modules/mob/living/carbon/human/species/skeleton.dm

```diff
@@ -10,6 +10,9 @@
 	default_language = "Galactic Common"
 	unarmed_type = /datum/unarmed_attack/punch
 
+	greater_form = null
```

Comment:
this is only used for monkies, so this is unnecessary and is already null, I believe.